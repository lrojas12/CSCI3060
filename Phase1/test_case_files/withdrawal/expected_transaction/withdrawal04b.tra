10 Omar                 00011          S 
00 Omar                 00011          S 