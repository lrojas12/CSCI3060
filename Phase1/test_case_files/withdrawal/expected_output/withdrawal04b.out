 __    __      _                           _             
/ / /\ \ \__ _| |_ ___ _ __ _ __ ___   ___| | ___  _ __  
\ \/  \/ / _` | __/ _ \ '__| '_ ` _ \ / _ \ |/ _ \| '_ \ 
 \  /\  / (_| | ||  __/ |  | | | | | |  __/ | (_) | | | |
  \/  \/ \__,_|\__\___|_|  |_| |_| |_|\___|_|\___/|_| |_|
  
Welcome to Watermelon Banking System. 
Please log in to begin or enter "help" for more information.

Enter mode in which you wish to log in as: 

Enter account holder's name:

You are currently logged in as Omar
BankAccount Number: 00011
Balance: $01000.00
Transaction PaymentPlan: Non-student
Status: Disabled

Enter a command.

Withdrawal transaction selected.

Enter account number: 

>>> ERROR: The account is disabled; you may not withdraw funds.

You have successfully logged out of your account.

Login or enter "help" for more information.